File: atcoder/abc351a_the_bottom_of_the_ninth.hs
-- vicfred
-- https://atcoder.jp/contests/abc351/tasks/abc351_a
-- implementation
main :: IO ()
main = do
  a <- map read . words <$> getLine :: IO [Int]
  b <- map read . words <$> getLine :: IO [Int]
  print $ (sum a) - (sum b) + 1

File: atcoder/abc351a_the_bottom_of_the_ninth.cpp
// vicfred
// https://atcoder.jp/contests/abc351/tasks/abc351_a
// math
#include <iostream>

using namespace std;

int main() {
  int a = 0;
  for(int i = 0; i < 9; ++i) {
    int ins;
    cin >> ins;
    a += ins;
  }
  int b = 0;
  for(int i = 0; i < 8; ++i) {
    int ins;
    cin >> ins;
    b += ins;
  }
  int res = a - b;
  res += 1;
  cout << res << endl;
}
